### ISSUE TITLE
Website menu disappears if page is not full width

### ISSUE THREAD

**Issue by HeidiSeibold** [rse-asia/RSE_Asia#46]
This is what the website looks like for me on full screen mode:

![Screenshot 2022-10-19 at 14 40 12](https://user-images.githubusercontent.com/14146757/196693362-e54d0e86-2772-4c07-b173-eac7c9e0018f.png)

However if I decrease the width of my browser window it looks like this:

![image](https://user-images.githubusercontent.com/14146757/196693593-923f3faf-8d68-452c-9562-69fcb0570b01.png)

Suddenly the menu disappears.

What I would expect to happen is a [hamburger button](https://en.wikipedia.org/wiki/Hamburger_button) to appear, but that does not seem to be the case here.

**Comment by SaranjeetKaur** [comment]
Hi @HeidiSeibold, thanks for reporting this! Coincidentally, I observed this while collaborating on the website work with @jyoti-bhogal today! I will have a deeper look at it and fix the issue.

**Comment by joelnitta** [comment]
Just an idea... have you considered migrating to [quarto](https://quarto.org/docs/websites/)? It has a very similar layout to distill, but many features are easier to use. It is possible this problem might go away (or might not! but worth a try IMHO).

**Comment by SaranjeetKaur** [comment]
Yes, we are planning to migrate to Quarto @joelnitta. But it will take some time
